Gem/Code/Source/CommonSampleComponentBase.h

```diff
@@ -114,9 +114,6 @@ namespace AtomSampleViewer
         //! Lights created by lighting presets.
         AZStd::vector<AZ::Render::DirectionalLightFeatureProcessorInterface::LightHandle> m_lightHandles;
 
-        //! Post process entity to handle ExposureControlSettings.
-        AZ::Entity* m_postProcessEntity = nullptr;
-

```

Comment:
This member variable is used here?
https://github.com/o3de/o3de-atom-sampleviewer/blob/be4859eba6255d4302effa8e7138e8147b44f890/Gem/Code/Source/CommonSampleComponentBase.cpp#L82

Reply:
Solved, I was looking at the development branch, change is for stabilization.